specs/interop/predeploys.md

```diff
@@ -424,6 +435,8 @@ function relayMessage(ICrossL2Inbox.Identifier calldata _id, bytes calldata _sen
 
 Note that the `relayMessage` function is `payable` to enable relayers to earn in the gas paying asset.
 
+Additionally, if the `_entrypoint` is not `address(0)` and the caller is not `_entrypoint`, the message must not be relayed.
```

Comment:
maybe "must revert" or "cannot be relayed" instead of "must not be relayed". Otherwise, it seems like its possible.

Reply:
'is going to revert' to affirm what should happen?

Reply:
yes, thats perfect :)